README

```diff
@@ -87,10 +87,11 @@ EXAMPLES
      As usual with pipes, the sky is the limit.
 
 CONCEPTS
-     mblaze deals with messages (which are files), folders (which are Maildir
-     folders), sequences (which are newline-separated lists of messages,
-     possibly persisted on disk in ~/.mblaze/seq), and the current message
-     (kept as a symlink in ~/.mblaze/cur).
+     mblaze deals with messages (which are files), folders (which are
```

Comment:
This file is generated by "make README", if you have mandoc installed... else don't change it and i'll update it then.